DeviceConnectionAndReliabilityReadme.md

```diff
@@ -0,0 +1,63 @@
+# Azure IoT Device Client .NET SDK
+
+## Device connection and messaging reliability
+
+### Overview
+
+In this document you will find information about:
+
+- The connection authentication and renewal methods.
+- The reconnection logic and retry policies.
+- The timeout controls.
+
+### Connection authentication
+
+Authentication can be done using either:
```

Comment:
The word "either" implies 2 options. A better choice would be "one of the following"

```suggestion
Authentication can be done using one of the following options:
```